blog/en/blog/2023/06/30/release-apache-apisix-3.4.0.md

```diff
@@ -0,0 +1,112 @@
+---
+title: "Release Apache APISIX 3.4.0"
+authors:
+  - name: "Xin Rong"
+    title: "Author"
+    url: "https://github.com/AlinsRan"
+    image_url: "https://avatars.githubusercontent.com/u/79972061?v=4"
+  - name: "Traky Deng"
+    title: "Technical Writer"
+    url: "https://github.com/kayx23"
+    image_url: "https://avatars.githubusercontent.com/u/39619599?v=4"
+keywords:
+- Apache APISIX
+- API Gateway
+- API Management Platform
+- New Release
+- Cloud Native
+description: The Apache APISIX 3.4.0 version is released on June 30, 2023. This version adds a new plugin for Grafana Loki, allows for mTLS connection on the route level, and made performance optimization to continuously enhance the user experience of APISIX.
+tags: [Community]
+---
+
+We are pleased to present Apache APISIX 3.4.0 with exciting new features and performance improvements.
+
+This release provides a new plugin `loki-logger` to forward logs to [Grafana Loki](https://grafana.com/oss/loki/), and allows for mTLS connection on the route level. In addition, the release also includes many other updates to continuously enhance the user experience of APISIX.
```

Comment:
Please add `<!--truncate-->` to avoid display issue in the blog summary page. An example of missing  `<!--truncate-->` would be https://github.com/apache/apisix-website/issues/1606, this issue will give you a better idea.
```suggestion
We are pleased to present Apache APISIX 3.4.0 with exciting new features and performance improvements.

<!--truncate-->

This release provides a new plugin `loki-logger` to forward logs to [Grafana Loki](https://grafana.com/oss/loki/), and allows for mTLS connection on the route level. In addition, the release also includes many other updates to continuously enhance the user experience of APISIX.
```